adapters/mobfoxpb/mobfoxpb_test.go

```diff
@@ -0,0 +1,15 @@
+package mobfoxpb
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, _ := Builder(openrtb_ext.BidderMobfoxpb, config.Adapter{
+		Endpoint: "http://example.com/?c=o&m=ortb"})
+	adapterstest.RunJSONBidderTest(t, "mobfoxpbtest", bidder)
+}
```

Comment:
I understand that it's not possible for your adapter to throw an error, but I'd like to ask for you to still please include the error check for uniformity with other adapter and in case that changes in the future. 

```
func TestJsonSamples(t *testing.T) {
func TestJsonSamples(t *testing.T) {
	bidder, buildErr := Builder(openrtb_ext. BidderMobfoxpb, config.Adapter{
		Endpoint: "http://example.com/?c=o&m=ortb"})

	if buildErr != nil {
		t.Fatalf("Builder returned unexpected error %v", buildErr)
	}

	adapterstest.RunJSONBidderTest(t, "mobfoxpbtest", bidder)
}
}
```

Reply:
done